setup/setup-scripts/install-python3-plus-libraries.bash

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+#1) Install python 3:
+sudo apt install python3.7 python-pip -y
```

Comment:
The scripts can't contain sudo commands; these will not work, for example, on openlab. Instead, it's best to instruct the user with an error message to find the required version of Python. On openlab they'd probably use the "module load" command, and elsewhere it's up to them.

Reply:
Got it.

I just copied the format from the existing file "install-python2-plus-libraries.bash". Shall I modify these two files？